xwiki-platform-core/xwiki-platform-web/xwiki-platform-web-war/src/main/webapp/resources/uicomponents/widgets/list/xlist.js

```diff
@@ -98,7 +98,8 @@ var XWiki = function(XWiki){
           this.options = options || {};
           var classes = 'xitem ' + (this.options.noHighlight ? '' : 'xhighlight ');
           classes += this.options.classes ? this.options.classes: '';
-          this.containerElement = new Element("div", {'class': 'xitemcontainer'}).insert(content || '');
+          var containerNature = this.options.containerNature ? this.options.containerNature: 'div';
```

Comment:
```suggestion
          var containerNature = this.options.containerNature || 'div';
```

But I'm not sure about the "Nature" part. I haven't seen this term used when talking about DOM and HTML. Maybe ``containerTagName`` is more explicit?

Reply:
I changed it to `containerTagName` in the last commit :+1: 
I was not sure what to call it, I also thought about `containerType` but I couldn't really find a name for this string when looking online.